voila/__init__.py

```diff
@@ -10,6 +10,9 @@
 from ._version import __version__  # noqa
 from .server_extension import _load_jupyter_server_extension  # noqa
 from .server_extension import load_jupyter_server_extension  # noqa
+import warnings
+
+warnings.filterwarnings("default", category=DeprecationWarning, module="traitlets")
```

Comment:
Mmmh, Maybe we should not do that and fix the deprecation warnings instead?

Reply:
Nope, the `DeprecationWarning` is hidden by default https://docs.python.org/3/library/exceptions.html#DeprecationWarning